src/main/scala/viper/gobra/translator/encodings/adts/AdtEncoding.scala

```diff
@@ -300,7 +300,7 @@ class AdtEncoding extends LeafTypeEncoding {
       formalArgs = Seq(variableDecl),
       typ = vpr.Int,
       unique = false,
-      interpretation = None
+      //interpretation = None
```

Comment:
At the time, when I tried to compile it, it gives some kind of error that there is an extra line in the AdtEncoding. I am not too sure, but I think it was because of the version of gobra that I used, so I changed it to make it compile.

Reply:
Make sure that your Viperserver folder is up to date. The interpretation node is newer addition and was not there in older versions.